workers/unity/Packages/io.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -38,7 +40,15 @@ private static void BuildConfigMenu()
         private static void LaunchMenu()
         {
             Debug.Log("Launching SpatialOS locally...");
-            EditorApplication.delayCall += LaunchLocalDeployment;
+            EditorApplication.delayCall += ConfigureLocalDeployment;
+        }
+
+        [MenuItem("SpatialOS/Local launch with custom snapshot", false, MenuPriorities.LocalLaunch)]
```

Comment:
MenuPriority will clash with LocalLaunch, unsure what the ordering will be now?

Reply:
As a design question. Do we want this to be 2 separate launcher, or just 1 launch that always pick whatever snapshot you've set?

Reply:
"As a design question. Do we want this to be 2 separate launcher, or just 1 launch that always pick whatever snapshot you've set?"

We could have one "Local Launch" button that would simply load whichever snapshot you've selected, since the parameter is populated with the default snapshot anyway.

The idea was to explicitly show the custom snapshot option, but I can change it. Also having only one options would fix the MenuPriority issue.

Reply:
I think that would be the best option